extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -126,49 +149,271 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 
 	public async populateMigrationController(controllerStatus?: string): Promise<void> {
 		this.migrationControllerDropdown.loading = true;
-		let migrationContollerValues: azdata.CategoryValue[] = [];
+		try {
+			this.migrationControllerDropdown.values = await this.migrationStateModel.getMigrationControllerValues(this.migrationStateModel._targetSubscription, this.migrationStateModel._targetManagedInstance);
+			if (this.migrationStateModel.migrationController) {
+				this.migrationControllerDropdown.value = {
+					name: this.migrationStateModel.migrationController.id,
+					displayName: `${this.migrationStateModel.migrationController.name}`
+				};
+			} else {
+				this.migrationStateModel.migrationController = this.migrationStateModel.getMigrationController(0);
+			}
+		} catch (error) {
+			console.log(error);
+		} finally {
+			this.migrationControllerDropdown.loading = false;
+		}
 
-		// TODO: Replace with this code when APIs are deployed.
-		// try{
-		// 	this.migrationControllerDropdown.values = await this.migrationStateModel.getMigrationControllerValues(this.migrationStateModel._targetSubscription, this.migrationStateModel._targetManagedInstance);
-		// 	this.migrationStateModel.migrationController = this.migrationStateModel.getMigrationController(0);
-		// } catch (e) {
+	}
 
-		// } finally {
-		// 	this.migrationControllerDropdown.loading = false;
-		// }
+	private async loadControllerStatus(): Promise<void> {
+		this._statusLoadingComponent.loading = true;
```

Comment:
Not doing a try/finally to ensure that the loading component goes away if an error occurs?